README.md

```diff
@@ -1,46 +1,95 @@
+
+For other systems, see the documentation at https://php.net/install.
+
+### Building PHP from source
+
+On \*nix systems:
 
-New features require an RFC and must be accepted by the developers.
-See https://wiki.php.net/rfc and https://wiki.php.net/rfc/voting for more
+    ./buildconf
+    ./configure
+    make
+    make test
+    sudo make install
+
+For Windows see https://wiki.php.net/internals/windows/stepbystepbuild_sdk_2.
+
+## PHP extensions
+
+Extensions provide additional functionality on top of PHP. PHP consists of many
+essential bundled extensions. Additional extensions can be found in the PHP
+Extension Community Library (PECL) at https://pecl.php.net.
+
+## Contributing
+
+The PHP source code is located in the Git repository at https://git.php.net.
+Contributions are most welcome by forking the GitHub mirror repository
+https://github.com/php/php-src and sending a pull request.
+
+Discussions are done on GitHub, but depending on the topic can also be relayed
+to the official PHP developer mailing list internals@lists.php.net.
+
+New features require an RFC and must be accepted by the developers. See
+https://wiki.php.net/rfc and https://wiki.php.net/rfc/voting for more
 information on the process.
 
-Bug fixes **do not** require an RFC, but require a bugtracker ticket. Always
-open a ticket at https://bugs.php.net and reference the bug id using #NNNNNN.
+Bug fixes **do not** require an RFC, but require a bug tracker ticket. Open a
+ticket at https://bugs.php.net and reference the bug id using #NNNNNN.
 
     Fix #55371: get_magic_quotes_gpc() throws deprecation warning
 
-    After removing magic quotes, the get_magic_quotes_gpc function caused
-    a deprecate warning. get_magic_quotes_gpc can be used to detected
-    the magic_quotes behavior and therefore should not raise a warning at any
-    time. The patch removes this warning
+    After removing magic quotes, the get_magic_quotes_gpc function caused a
+    deprecated warning. get_magic_quotes_gpc can be used to detect the
+    magic_quotes behavior and therefore should not raise a warning at any time.
+    The patch removes this warning.
+
+Pull requests are not merged directly on GitHub. All PRs will be pulled and
+pushed through https://git.php.net.
 
-We do not merge pull requests directly on github. All PRs will be
-pulled and pushed through https://git.php.net.
+### Guidelines for contributors
 
+See further documents in the repository for more information how to contribute:
 
-Guidelines for contributors
-===========================
+- [CONTRIBUTING.md](/CONTRIBUTING.md)
 - [CODING_STANDARDS](/CODING_STANDARDS)
 - [README.GIT-RULES](/README.GIT-RULES)
 - [README.MAILINGLIST_RULES](/README.MAILINGLIST_RULES)
 - [README.RELEASE_PROCESS](/README.RELEASE_PROCESS)
 
 ## Testing
 
-To run tests the `make test` is used after successful compilation of the sources.
+To run tests the `make test` is used after a successful compilation of the
+sources.
```

Comment:
Also believe is okay to add a note about the `-j` option that we introduced at #3838.

Reply:
Also added...